extensions/warp-ipfs/examples/messenger.rs

```diff
@@ -369,6 +370,7 @@ async fn main() -> anyhow::Result<()> {
                             let topic = *topic.read();
                             let mut settings = GroupSettings::default();
                             settings.set_members_can_add_participants(open);
+                            settings.set_members_can_change_name(open);
```

Comment:
This should probably be under its own command to make it more consistent with the rest of the code

Reply:
Well, we opted for the very generic "open" concept here so it would be strange to put it in a separate command. We can think about splitting this "open" concept into its individual parts in a separate PR for sure, though. We'll want to change the name of `--open` option of `create-group` as well then and add another one for this.